paddle/fluid/platform/temporary_allocator.cc

```diff
@@ -15,8 +15,10 @@
 #include "paddle/fluid/platform/temporary_allocator.h"
 #include "paddle/fluid/memory/allocation/allocator_facade.h"
 
-DEFINE_double(limit_of_temporary_allocation, -1,
-              "The up limit of temporary_allocation size.");
+DEFINE_int64(limit_of_tmp_allocation, -1,
+             "The up limit of temporary_allocation size.");
+DEFINE_double(reuse_tmp_allocation_excess_fraction, 2,
```

Comment:
First, I am not sure the best value of excess fraction.

`reuse_tmp_allocation_excess_fraction` indicates the max memory size that `TemporaryAllocator` can return. 

This is to say, if user want to get 4KB from `TemporaryAllocator`, but the available memory of `TemporaryAllocator` only have 3KB, 4GB, we should not return 4GB generally, because it is too waste memory. 
If the available memory is 3KB, 6KB, 9KB, we should return 6KB; if user want to get the exacted equal size from `TemporaryAllocator`, he should set `reuse_tmp_allocation_excess_fraction` below 1.0  **this is why the fraction should be larger than 1.0.**